frontend/model/captureLogs.js

```diff
@@ -61,7 +61,16 @@ function captureLogEntry (type, ...args) {
     msg: args.map((arg) => {
       try {
         return JSON.parse(
-          JSON.stringify(arg instanceof Error ? (arg.stack ?? arg.message) : arg)
+          JSON.stringify(arg, (_, v) => {
+            if (v instanceof Error) {
+              return {
+                name: v.name,
+                message: v.message,
+                stack: v.stack
+              }
+            }
+            return v
+          })
```

Comment:
Why did you do it this way instead of sticking with the previous `?:` method?

Reply:
This improves the logs by adding more information about errors, and it also captures errors anywhere, not just as an argument. When debugging, I needed to see the message and error name as well as the stack.